RELEASE-NOTES.rst

```diff
@@ -1,8 +1,8 @@
-=======================
- Invenio-I18N v1.0.0b3
-=======================
+======================
+ Invenio-I18N v1.0.0b4
+======================
 
-Invenio-I18N v1.0.0b3 was released on February 17, 2017.
+Invenio-I18N v1.0.0b4 was released on Aug 09, 2017.
```

Comment:
I have not seen any release note where we used abbreviated months. IMHO we should use "August" just to be consistent with the other releases.